pyogrio/core.py

```diff
@@ -218,3 +219,13 @@ def get_gdal_config_option(name):
     """
 
     return _get_gdal_config_option(name)
+
+
+def get_gdal_data_path():
+    """Get the path to the directory GDAL uses to read data files.
+
+    Returns
+    -------
+    str, or None if data directory was not found
+    """
+    return _get_gdal_data_path()
```

Comment:
I took this approach of wrapping into another function to mimic what we did for `get/set_gdal_config_option` (although it seems a bit superfluous)